### PR TITLE
Move Notifications to its own library

### PR DESCRIPTION
**Pull request by Carbenium**

### CODE REVIEW

File: src/Libraries/Notifications/Notifications/Notification.cs [hunk 1]
```diff
@@ -32,39 +32,40 @@
 using EventArgs = System.EventArgs;
 
 namespace Notifications {
```
Comment (arfbtwn) [inline src/Libraries/Notifications/Notifications/Notification.cs]: I thought the enum braces were wrong when I first looked but it's the namespaces and it's everywhere in the codebase, oh the horror! :)

I am kidding - but do we need an addition to the R# style rules?

Comment (Carbenium) [reply src/Libraries/Notifications/Notifications/Notification.cs]: Good catch :wink: 

R# settings seem fine. I just didn't run the auto formatting.

**Comment by Carbenium** [comment]
Alright. I just ran the code cleanup using our rules (PR with couple of changes incoming).

The reviewing experience might just have become worse but let's see what you say :wink: 
If you're okay with the last commit (06ad6f8) I'm gonna squash it with the previous.

**Comment by arfbtwn** [comment]
I gave it a slight massage and reduced it to the new library addition (all in) and then the auto-formatting, only thing I did to that was restore some line separation to the using declarations.

Since you're away I pushed this out and created a fresh pull-request, #40, if you're happy with it chuck an approval on and I'll merge it in.

**Comment by arfbtwn** [comment]
merged #40, cheers

**Pull request closed without merging**